### ISSUE TITLE
Error: No type found for .Point

### ISSUE THREAD

**Issue by alecthomas** [stephenh/ts-proto#6]
This proto file:

```protobuf
syntax = "proto3";

message Point {
    double lat = 1;
    double lng = 2;
}

message Area {
    Point nw = 1;
    Point se = 2;
}
```

with this command:

```
protoc --plugin=../../node_modules/ts-proto/protoc-gen-ts_proto --ts_proto_out=. -I../../../protos ../../../protos/api.proto
```

results in this error:

```
FAILED!No type found for .PointError: No type found for .Point
    at Object.fail (/Users/alec/Library/Caches/realestate/node_modules/ts-proto/build/utils.js:21:11)
    at toModuleAndType (/Users/alec/Library/Caches/realestate/node_modules/ts-proto/build/types.js:256:59)
    at messageToTypeName (/Users/alec/Library/Caches/realestate/node_modules/ts-proto/build/types.js:250:28)
    at basicTypeName (/Users/alec/Library/Caches/realestate/node_modules/ts-proto/build/types.js:81:20)
    at Object.toTypeName (/Users/alec/Library/Caches/realestate/node_modules/ts-proto/build/types.js:260:16)
    at generateInterfaceDeclaration (/Users/alec/Library/Caches/realestate/node_modules/ts-proto/build/main.js:178:107)
    at /Users/alec/Library/Caches/realestate/node_modules/ts-proto/build/main.js:31:34
    at visit (/Users/alec/Library/Caches/realestate/node_modules/ts-proto/build/main.js:201:9)
    at Object.generateFile (/Users/alec/Library/Caches/realestate/node_modules/ts-proto/build/main.js:30:5)
    at /Users/alec/Library/Caches/realestate/node_modules/ts-proto/build/plugin.js:18:29--ts_proto_out: protoc-gen-ts_proto: Plugin failed with status code 1.
```

**Comment by alecthomas** [comment]
Looks like I'm on 1.4.1:
```
package.json:    "ts-proto": "^1.4.1",
```

**Comment by stephenh** [comment]
@alecthomas thanks for the report! This should be fixed in 1.4.2. The issue was I didn't have coverage of proto files that don't have the `package` value set. We'd always used that, so I didn't realize it was optional. Let me know if that doesn't work or if you run into anything else.

**Comment by alecthomas** [comment]
Champion!
